<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateDataPlumeDialog</class>
 <widget class="QDialog" name="CreateDataPlumeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>626</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Data Plume</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;After submitting by clicking Create Plume, your data reservation request will be sent to available Neural Nodes if you have chosen to use them.  Please wait for data reservation proposals to come back, choose the ones you want and Accept them to finalize the creation of your Data Plume.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>9</number>
       </property>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="nameLineEdit"/>
       </item>
       <item>
        <widget class="QCheckBox" name="plumeTypeCheckBox">
         <property name="text">
          <string>Public</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Attribute 1 Name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="attrOneLineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Attribute 2 Name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="attrTwoLineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Attribute 3 Name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="attrThreeLineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Kilobyte-Hours to Reserve:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="kbhSpinBox">
         <property name="maximum">
          <double>9999999999.989999771118164</double>
         </property>
         <property name="singleStep">
          <double>1000.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string># of Neural Nodes (0 for local only):</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="slavesSpinBox">
         <property name="value">
          <number>8</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Neural Node Data Reservation Proposals:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="proposalsTableWidget">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>175</number>
       </attribute>
       <column>
        <property name="text">
         <string>Peer Id</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Rate (XAI/kilobyte-hour)</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Cost (XAI)</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Payment Address</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Proposal Id</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="peersLabel">
         <property name="text">
          <string>0 Plume peers connected</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="createPlumeButton">
         <property name="text">
          <string>Create &amp;Plume</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="acceptProposalsButton">
         <property name="text">
          <string>&amp;Accept Proposals</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelButton">
         <property name="text">
          <string>&amp;Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
